### PR TITLE
Modify pipeline for private build

### PR DESCRIPTION
**Pull request by kennep**

### CODE REVIEW

File: Directory.Build.props [hunk 1]
```diff
@@ -2,8 +2,7 @@
   <!-- Package Metadata -->
   <PropertyGroup>
     <VersionPrefix>1.1.0</VersionPrefix>
-    <VersionSuffix>
-    </VersionSuffix>
+    <VersionSuffix>vipps-1</VersionSuffix>
```
Comment (estensen) [inline Directory.Build.props]: why `vipps-1`?

File: azure-pipelines.yml [hunk 2]
```diff
@@ -69,21 +77,9 @@ jobs:
...
-      reports: '$(Build.SourcesDirectory)/Test/coverage.netcoreapp3.1.cobertura.xml'
-      targetdir: '$(Build.SourcesDirectory)/CodeCoverage'
-      reporttypes: 'HtmlInline_AzurePipelines;Cobertura;Badges'
-      assemblyfilters: '-xunit*'
   - task: PublishCodeCoverageResults@1
     displayName: 'Publish code coverage'
     inputs:
       codeCoverageTool: Cobertura
       summaryFileLocation: '$(Build.SourcesDirectory)/CodeCoverage/Cobertura.xml'
       reportDirectory: '$(Build.SourcesDirectory)/CodeCoverage'
```
Comment (estensen) [inline azure-pipelines.yml]: Should we remove this as well?

File: Directory.Build.props [hunk 3]
```diff
@@ -2,8 +2,7 @@
...
     <Description>FIDO2 .NET library (WebAuthn)</Description>
     <RepositoryUrl>https://github.com/abergs/fido2-net-lib</RepositoryUrl>
```
Comment (estensen) [inline Directory.Build.props:8]: update URL?

**Review by estensen — APPROVED** [review]

File: Directory.Build.props (hunk 1 above)
Comment (kennep) [reply Directory.Build.props]: Remnants from an earlier experiment. Removing.

File: azure-pipelines.yml (hunk 2 above)
Comment (kennep) [reply azure-pipelines.yml]: Yes.

File: Directory.Build.props (hunk 3 above)
Comment (kennep) [reply Directory.Build.props:8]: OK, done.

**Pull request merged**